examples/tts/speechllm/conf/megatron_t5_speechllm_inference_model.yaml

```diff
@@ -54,8 +54,6 @@ model:
   validation_metric: accuracy
   num_speech_tokens: 10112 # Vocabulary size pertaining to speech
   seq_pattern: "parallel" # parallel, delay_parallel, flatten
-  speech_head_type: "linear" # token_level, linear
-  cross_entropy_type: "vocab_parallel" # regular, vocab_parallel
   temperature: 0.7 # Temperature to be used for inference
   top_k: 80 # Top k to be used for inference
```

Comment:
In the code, the defaults are `temperature = self.cfg.get('temperature', 0.85)  # Set temp 0.01 for greedy decoding`, should we update the configs to match?

Reply:
Yes, makes sense. Let me do that